src/include/duckdb/main/capi/header_generation/apis/v1/v1.2/v1.2.0.json

```diff
@@ -193,13 +271,21 @@
         "duckdb_replacement_scan_set_function_name",
         "duckdb_replacement_scan_add_parameter",
         "duckdb_replacement_scan_set_error",
+        "duckdb_get_profiling_info",
+        "duckdb_profiling_info_get_value",
+        "duckdb_profiling_info_get_metrics",
+        "duckdb_profiling_info_get_child_count",
+        "duckdb_profiling_info_get_child",
         "duckdb_appender_create",
+        "duckdb_appender_create_ext",
         "duckdb_appender_column_count",
         "duckdb_appender_column_type",
         "duckdb_appender_error",
         "duckdb_appender_flush",
         "duckdb_appender_close",
         "duckdb_appender_destroy",
+        "duckdb_appender_add_column",
+        "duckdb_appender_clear_columns",
```

Comment:
Hmm, I'd be more careful about these. For appending a few rows, or a data chunk with sparse default values, this might be relevant. That is, maybe we want to keep a slim row-based append functionality. Have a look at my comment here: https://github.com/duckdb/duckdb/pull/15121#discussion_r1869129779